Makefile

```diff
@@ -987,7 +987,7 @@ endif
 
 ifdef SANE_TOOL_PATH
 SANE_TOOL_PATH_SQ = $(subst ','\'',$(SANE_TOOL_PATH))
-BROKEN_PATH_FIX = 's|^\# @@BROKEN_PATH_FIX@@$$|git_broken_path_fix $(SANE_TOOL_PATH_SQ)|'
+BROKEN_PATH_FIX = "s|^\# @@BROKEN_PATH_FIX@@$$|git_broken_path_fix '$(SANE_TOOL_PATH_SQ)'|"
```

Comment:
We probably should use `\\#` instead of `\#` now, too (even if it seems to be accepted by Bash).